blackbox/docs/sql/reference/select.txt

```diff
@@ -238,6 +239,30 @@ values whitin a resulting row of a group by clause.
   Grouping can only be applied on indexed fields. For more information,
   please refer to :ref:`sql_ddl_index_off`.
 
+.. _sql_reference_union:
+
+UNION ALL Clause
+................
+
+The UNION ALL operator computes the set union of the rows returned by
+the involved SELECT statements. The two SELECT statements that represent
+the direct operands of the UNION ALL must produce the same number
+of columns, and corresponding columns must be of compatible data types.
```

Comment:
@dobe I was thinking of that and compatible is what Postgres does. Shouldn't we also try to cast for example int to long? Otherwise I'll change the the phrase to same.